_stbt/core.py

```diff
@@ -1999,25 +2006,20 @@ def on_new_sample(self, appsink):
         return Gst.FlowReturn.OK
 
     def tell_user_thread(self, frame_or_exception):
-        # `self.last_sample` (a synchronised Queue) is how we communicate from
-        # this thread (the GLib main loop) to the main application thread
-        # running the user's script. Note that only this thread writes to the
-        # Queue.
+        # `self.last_frame` is how we communicate from this thread (the GLib
+        # main loop) to the main application thread running the user's script.
+        # Note that only this thread writes to the Queue.
```

Comment:
only this thread writes to ~~the Queue~~ \`self.last_frame`.

Reply:
Fixed